client/reader/stream/index.jsx

```diff
@@ -186,6 +184,10 @@ export default class ReaderStream extends React.Component {
 	}
 
 	cardClassForPost = ( post ) => {
+		if ( includes( this.props.blockedSites, +post.site_ID ) ) {
```

Comment:
if we ever had the need for different streams to display different formats for blocked cards we would need to do a bit of rejiggering but for now I think its aight